services/workflows-service/src/sentry/sentry.service.ts

```diff
@@ -39,7 +40,13 @@ export class SentryService {
   }
 
   private _setExtraData(scope: Sentry.Scope, request?: Request) {
-    scope.setExtra('reqId', this.cls.get('requestId'));
```

Comment:
Is `traceId` and `reqId` the same? As far as I know, they are not the same. One request can have many traces.

Reply:
yeah both are used as context for request id which changed to trace id 

they have one associated trace with multiple spans